docs/data/registry.json

```diff
@@ -350,6 +350,40 @@
       ]
     }
   },
+  "mysql": {
+    "description": "A manufacture-based container for MariaDB",
```

Comment:
Who is the audience of this? Isn't "manufacture-based" an internal detail?

Reply:
It's not publicly shown anywhere other than this file, but ya it's from our repo description. An artifact of trying to generate this file programmatically from GitLab I reckon - fine to remove if we want to.